misc/config/config.nydus.ref.yaml

```diff
@@ -61,3 +63,6 @@ converter:
   rules:
     # add suffix to tag of source image reference as target image reference
     - tag_suffix: -nydus-oci-ref
+    # add suffix to tag of remote cache reference, default cache size is 200, leave empty to disable remote cache.
```

Comment:
`# add suffix to tag of source image reference as remote cache reference, leave empty to disable remote cache.`

Reply:
fixed